person.rb

```diff
@@ -1,3 +1,5 @@
+require './nameable'
+
 class Person
```

Comment:
- [ ] Good job working on this but please kindly ensure that the Person class inherits from Nameable class